dff/pipeline/types.py

```diff
@@ -59,51 +64,45 @@ class ExtraHandlerType(Enum):
     Enum, representing wrapper type, pre- or postprocessing.
     The following types are supported:
 
-        - PREPROCESSING: wrapper function called before component
-        - POSTPROCESSING: wrapper function called after component
+    - PREPROCESSING: wrapper function called before component,
+    - POSTPROCESSING: wrapper function called after component.
     """
 
     UNDEFINED = auto()
     BEFORE = auto()
     AFTER = auto()
 
 
+PIPELINE_STATE_KEY = "PIPELINE"
 """
 PIPELINE: storage for services and groups execution status.
 Should be used in `ctx.framework_keys[PIPELINE_STATE_KEY]`.
 """
-PIPELINE_STATE_KEY = "PIPELINE"
 
 
+StartConditionCheckerFunction = Callable[[Context, Actor], bool]
```

Comment:
Add `typing_extenstion.TypeAlias` to this and other types:
```suggestion
StartConditionCheckerFunction: TypeAlias = Callable[[Context, Actor], bool]
```

Reply:
Why should we write `: TypeAlias` for StartConditionCheckerFunction?